### PR TITLE
File Structure Change + Code Cleanup 

### PR DESCRIPTION
**Pull request by Aetherall**
The goal here is to incrementally update the file structure in order to make the Accounts Suite structure more consistent.

Any help is welcome, as well as reviews and opinions

### CODE REVIEW

**Comment by Aetherall** [comment]
Yes sure, I am waiting for the tslint PRs to be merged first ;)
https://github.com/accounts-js/tslint-config-accounts/pull/3
https://github.com/accounts-js/tslint-config-accounts/pull/4

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@accounts/tslint-config-accounts": "0.0.6",
+    "@accounts/tslint-config-accounts": "^0.0.9",
```
Comment (pradel) [inline package.json]: can you keep strict deps?

Comment (Aetherall) [reply package.json]: yes sure !

**Comment by Aetherall** [comment]
Create new PR to reflect master changes

**Pull request closed without merging**